python/Makefile.am

```diff
@@ -24,3 +24,13 @@ check-local:
 .PHONY: pylint
 pylint:
 	pylint $(SUBDIRS)
+
+# Disabled warnings:
+# E501 - Line too long
+# E722 - Do not use bare except
+# F401 - Module imported but unused
```

Comment:
seems like we should be able to fix those last two

Reply:
We're using bare except in three places, two of which I think we probably want to catch everything - run a test and if anything goes wrong stop profiling; and run a scenario and if anything goes wrong clean up.  The third could be fixed.  The unused import is for stuff like `pacemaker/__init__.py` where we are re-exporting things.

Reply:
Added a patch to address the bare export (catch the one exception we care about in one place, and ignore the other two with a pragma), and updated the Makefile.am patch to only ignore unused imports in `__init__.py` files.